src/Http/Middleware/CsrfProtectionMiddleware.php

```diff
@@ -193,6 +193,26 @@ public function createToken(): string
         return $value . hash_hmac('sha1', $value, Security::getSalt());
     }
 
+    /**
+     * Verify CSRF token.
+     *
+     * @param string $token The CSRF token.
+     * @return bool
+     */
+    protected function _verifyToken(string $token): bool
+    {
+        if (strlen($token) <= self::TOKEN_VALUE_LENGTH) {
```

Comment:
should those be mb_ operations? the token input could be any user provided input right?